File: Client/Debug/PW Plugin/SRC2/Recovery/Browsers/IPassReader.cs
using System;
using System.Collections.Generic;

namespace Plugin.Browsers
{
	// Token: 0x0200000E RID: 14
	internal interface IPassReader
	{
		// Token: 0x0600006D RID: 109
		IEnumerable<CredentialModel> ReadPasswords();

		// Token: 0x17000012 RID: 18
		// (get) Token: 0x0600006E RID: 110
		string BrowserName { get; }
	}
}

File: Client/Debug/PW Plugin/SRC/Recovery/Browsers/IPassReader.vb
Imports System
Imports System.Collections.Generic

Namespace Plugin.Browsers
	' Token: 0x0200000E RID: 14
	Friend Interface IPassReader
		' Token: 0x0600006D RID: 109
		Function ReadPasswords() As IEnumerable(Of CredentialModel)

		' Token: 0x17000012 RID: 18
		' (get) Token: 0x0600006E RID: 110
		ReadOnly Property BrowserName As String
	End Interface
End Namespace
